libs/api/ebpf_api.cpp

```diff
@@ -2759,6 +2774,8 @@ ebpf_program_unload(_Inout_ struct bpf_program* program) noexcept
         program->fd = ebpf_fd_invalid;
     }
     if (program->handle != ebpf_handle_invalid) {
+        std::unique_lock lock(_ebpf_state_mutex);
+
```

Comment:
```suggestion
        std::unique_lock lock(_ebpf_state_mutex);
```
Following the pattern you set above, it seems this extra newline is unnecessary?